implementations/rust/ockam/ockam_abac/src/policy/storage/resource_repository.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{Resource, ResourceName};
+use ockam_core::async_trait;
+use ockam_core::compat::boxed::Box;
+use ockam_core::Result;
+
+/// This repository stores the relationship between resources and resource types.
```

Comment:
The comment could just say now "stores resources".

Reply:
so many comments and doc strings 🤯